### ISSUE TITLE
[Bug]: 1.18.2, failed to import 'world', already exists

### ISSUE THREAD

**Issue by Mute-camcorder** [Multiverse/Multiverse-Core#2773]
### `/mv version -p` output

https://paste.gg/92362b6978c649a9a6e5b7fde41a5cbf

### Server logs

startup: https://paste.gg/p/anonymous/da54b7d31d4043eba04250f617956393 
import command: https://paste.gg/p/anonymous/ed9770b0d07c413baaea8b4eac28808d

### Server Version

```shell
This server is running Paper version git-Paper-349 (MC: 1.18.2) (Implementing API version 1.18.2-R0.1-SNAPSHOT) (Git: b6a6544)
You are running the latest version
Previous version: git-Paper-344 (MC: 1.18.2)
```

### Bug Description

Firstly, thank you for Multiverse, I have a friend who admins a server and in his opinion its the coolest plugin he has ever installed, and I have to agree.

---

I am unable to import my main world with `/mv import world normal`. 

I have run a scan with [Minecraft-Region-Fixer's latest version](https://github.com/Fenixin/Minecraft-Region-Fixer/tree/bfafd378ceb65116e4ea48cab24f1e6394051978), which found no issues.

I've reproduced the issue in a fresh server running in docker, and with Multiverse-Core 4.3.2 from jenkins.

### Steps to reproduce

1. Setup a new paper server in docker using [Docker Minecraft Server](https://github.com/itzg/docker-minecraft-server) at version 1.18.2
2. Download Multiverse-Core 4.3.1 into `/data/plugins`
3. Start the server (errors on plugin intialisation
4. Try to import `world` with `/mv import world normal`

### Agreements

- [X] I have searched for and ensured there isn't already an open or resolved issue(s) regarding this.
- [X] I was able to reproduce my issue on a freshly setup and up-to-date server with the latest version of Multiverse plugins with no other plugins and with no kinds of other server or client mods.

**Comment by MCPhssthpok** [comment]
I don't have a lot to add except to say that I'm seeing the same problem and that it appears to have started with PaperMC 1.18.2 build #348 -  Fix duplicate world keys via createWorld, which was released yesterday.

**Comment by SlimeDog** [comment]
Confirmed with Paper 1.18.2-349.

**Comment by SlimeDog** [comment]
Confirmed fixed in Paper 1.18.2-350.
